term/fhir-term-loader/pom.xml

```diff
@@ -0,0 +1,114 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.ibm.fhir</groupId>
+        <artifactId>fhir-term-parent</artifactId>
+        <version>4.10.0-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <artifactId>fhir-term-loader</artifactId>
```

Comment:
do we expect this project to evolve beyond loading just the fhir-term-graph?
if not, we might want to use `fhir-term-graph-loader` instead.  I'm good either way thought.

Reply:
Changed to be more specific